styles/autocomplete.less

```diff
@@ -44,6 +44,10 @@ autocomplete-suggestion-list.select-list.popover-list {
   .suggestion-description-content {
     font-size: @font-size + 1px;
     font-family: @font-family;
+    max-height: 300px;
```

Comment:
Should it be even less, maybe max height of `200px`?

![image](https://user-images.githubusercontent.com/378023/28809964-c88df16e-76c1-11e7-977b-14836878e121.png)

I think the description is more meant as a short summary or hint and less for adding the actual documentation.